assets/css/amp-editor-story-blocks.css

```diff
@@ -150,24 +150,66 @@ div[data-amp-position="lower-third"] {
 	display: none;
 }
 
-.editor-selectors {
+.edit-post-visual-editor ul.editor-selectors {
 	position: absolute;
-	right: -150px;
+	right: -155px;
 	bottom: 0;
-	width: 100px;
+	width: 120px;
 	z-index: 80;
+	list-style-type: none;
+	padding: 0;
+	margin: 0;
 }
 
 .editor-selectors .component-editor__selector {
-	margin-bottom: 2px;
+	background: no-repeat left top;
+	padding: 2px 0 0 30px;
+	height: 40px;
+	margin-left: 5px;
 }
 
-.editor-selectors .component-editor__selector button {
-	cursor: pointer;
+.editor-selectors .component-editor__selector.template-fill {
+	background-image: url( '../images/grid-fill-inactive.svg' );
```

Comment:
In the Gutenberg Inserter, the SVG images are actually inline in the document as opposed to being included from a separate file as a background-image in CSS. I don't know if necessarily one way is better than the other in this case, but I wanted to mention that.